packages/react/src/components/PromoBanner/PromoBanner.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import LinkWithIcon from '../LinkWithIcon/LinkWithIcon';
+import markdownToHtml from '@carbon/ibmdotcom-utilities/es/utilities/markdownToHtml/markdownToHtml';
+import PropTypes from 'prop-types';
+import React from 'react';
+import settings from 'carbon-components/es/globals/js/settings';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+const PromoBanner = ({ banner, copy, cta }) =>
```

Comment:
@guilhermelMoraes can you use the `featureFlag` utility as a wrapper to this? Should be similar to how we created `CalloutData` (as an example).